doc/index.rst

```diff
@@ -190,6 +190,25 @@ Note that some of these currently tied only to a single option, but we still lea
 
 Note: You see some `*PKGS*` variables. These make things very flexible but in normal usecases you don't need to be bothered with them - just keep them blank.
 
+Use custom Docker images
+------------------------
+
+As you see in the `optional variables section <./index.rst#optional-environment-variables>`_, there are a few different ways to specify `Docker` image if you like. Here are some more detail:
+
+Pulling Docker image from an online hub
++++++++++++++++++++++++++++++++++++++++
+
+You can pull any `Docker` image by specifying in `DOCKER_IMAGE` variable, as long as the following requirement is ment:
+
+* `python-catkin-tools`, `python-pip`, `python-wstool`
+* `python-rosdep` including rosdep repository set-up
```

Comment:
rosdep repository is not required, but ROS repo set-up is.

Reply:
Updated.